ledger/src/blockstore.rs

```diff
@@ -3808,21 +3846,44 @@ where
     Ok(())
 }
 
+// Update the meta for a slot whose parent is known to be connected.
+// Returns a bool indicating whether this slot is now connected which would
+// indicate that it's children are also in need of update.
+fn update_child_from_connected_parent(meta: &mut SlotMeta) -> bool {
```

Comment:
I feel it's better to make this function a meta function.

Reply:
Done, moved stuff to `SlotMeta::set_parent_connected`